packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,737 @@
+| name        | String                                                                        | Human-readable name for the Rule, configurable during creation or updating.           |
+| id          | String                                                                        | Unique identifier for this Rule (read-only).                                          |
+| type        | String                                                                        | Type of the Rule. For Identifier Match Rule Objects, this needs to be `Okta:SignOn`.  |
+| priority    | Integer                                                                       | Used to determine which Rules take. precedence.                                       |
+| conditions  | Array                                                                         | UserType, User, Group and various Device conditions are supported for this rule type. |
+| action      | String                                                                        | Either `ALLOW` or `DENY`. Controls whether the user is allowed to proceed.            |
+| requirement | [User Profile Rule Requirement Object](#user-profile-rule-requirement-object) | Specifies profile attributes to prompt user for.                                      |
+| status      | String                                                                        | `ACTIVE`  or  `INACTIVE`.                                                             |
+| default     | Boolean                                                                       | `true` for the first instance of this Rule, which gets created by default.            |
+
+#### User Profile Rule Requirement Object
+
+| Property                   | Type   | Description                                                                                                                                                                                                                                    |
+|----------------------------|--------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| preRegistrationInlineHooks | String | Optional. The valid ID of an Inline Hook. Sets a Registration Inline Hook to invoke.                                                                                                                                                           |
+| profileAttributes          | Array  | Array specifying the `label` and `name` of each User Profile Attribute that should be collected from the user. `label` is human-readable text that can be displayed to the user in a web form. `name` is the name of a User Profile Attribute. |
```

Comment:
`required` is not mentioned but show up in sample. I'd assume it's obvious what does `required` mean but just double confirm whether it's intentional at omitting from description.

Reply:
Yes, should add `required` here also, with note to clarify than non-required fields may be required at user enrollment time based on the User Type's schema

Reply:
Changed to:
| profileAttributes          | Array | Array specifying the `label` and `name` of each User Profile Attribute that should be collected from the user, as well as a `required` property set to `true` or `false` . `label` is human-readable text that can be displayed to the user in a web form. `name` is the name of a User Profile Attribute. |

>**Note**: Even if a User Profile Attribute is not set to be required, it can be required at user enrollment time based on the User Type schema.